_rules/SC1-3-5-autocomplete-valid.md

```diff
@@ -0,0 +1,177 @@
+---
+name: Autocomplete valid
+description: |
+  This rule checks that the HTML autocomplete attribute has a correct value
+
+success_criterion:
+- 1.3.5 # Identify Input Purpose
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any HTML `input`, `select` and `textarea` element with a [non-empty](#non-empty) `autocomplete` attribute, except if one of the following is true:
+
+- The element is **not** [visible on the page](#visible-on-the-page) or [exposed to assistive technologies](#exposed-to-assistive-technologies)
```

Comment:
I think this sentence can be interpreted in two ways.  

Way 1: not (visible or exposed)
Way 2: (not visible) or (exposed)

This could be cleared up by making it two bullets.